testing/trino-tests/src/test/java/io/trino/execution/TestEventListenerBasic.java

```diff
@@ -1326,4 +1275,67 @@ private static String getQualifiedName(TableInfo tableInfo)
     {
         return tableInfo.getCatalog() + '.' + tableInfo.getSchema() + '.' + tableInfo.getTable();
     }
+
+    static class Assertions
+    {
+        static TableInfoAssert assertThat(TableInfo actual)
+        {
+            return new TableInfoAssert(actual);
+        }
+    }
+
+    static class TableInfoAssert
```

Comment:
nit: We could move it to a dedicated test class

Reply:
sounds good, moved everything into a new package `io.trino.common.assertions`